setup/azurecmdfiles.wxi

```diff
@@ -44,15 +44,6 @@
                         <Component Id="cmpDC433AC0D6833219D86CA13947A3934D" Guid="*">
                             <File Id="filA95ABA71ABB678931802000F97D03EDA" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\ApiManagement\Microsoft.Azure.ResourceManager.dll" />
                         </Component>
-                        <Component Id="cmp1A6CF5E9ED315E384B7AB1184EC3175B" Guid="*">
```

Comment:
These deletions concern me - they are not in your project, I'm curious why this occurred.  If you rebuild and build the installer, these should not be removed